source/auth/auth.rst

```diff
@@ -1364,6 +1367,21 @@ interpret it as a file path.  The contents of the file are read as the
 access token.  If the path does not exist or cannot be read, or the environment variable does not exist, the driver MUST raise an error.
 
 
+AZURE
+_____
+When the PROVIDER_NAME mechanism property is set to "azure", the driver MUST
+attempt to retrieve a credential from the Azure Instance Metadata Service (IMDS).
+The driver MUST ensure that the TOKEN_AUDIENCE mechanism property is set when
+the PROVIDER_NAME is "azure".
+
+The driver MUST use the same procedure given in `Obtaining an Access Token for Azure Key Vault`_ to
+obtain an access token, with the one exception of using the TOKEN_AUDIENCE as
+the ``resource`` parameter.  The azure credentials MUST be cached by
+TOKEN_AUDIENCE, and expire within 5 minutes of the time given by the
+``expires_in`` parameter of the IMDS response.
```

Comment:
I think we should call out that the `resource=https://vault.azure.net/` query parameter in the URI generated in the other link should be changed to `resource=api://${AZUREOIDC_CLIENTID}` and may in some cases need toURL encode the `://` characters.

Reply:
I think it would be more accurate to say that the audience itself is an encoded URI, most likely an Application ID URI of the form `app://<app id>`.